.release-notes/106.md

```diff
@@ -0,0 +1,38 @@
+## Change HTTPClient handler factory creation
+
+The handler factory for creating handlers for new requests is now provided
+in the constructor of the client instead of the apply method. This
+makes it more clear, that the client will use the same handler for all
+requests.
```

Comment:
```suggestion
The handler factory for creating handlers for new requests is now provided in the constructor of the client instead of the apply method. This makes it more clear, that the client will use the same handler for all requests.
```